distributed/tests/test_joblib.py

```diff
@@ -71,3 +69,73 @@ def test_joblib_funcname(joblib):
     func = BatchedCalls([(random2,), (random2,)])
     assert joblib_funcname(func) == 'random2'
     assert joblib_funcname(random2) == 'random2'
+
+
+@pytest.mark.parametrize('joblib', joblibs)
+def test_joblib_backend_subclass(joblib):
+    if joblib is None:
+        pytest.skip()
+
+    assert issubclass(distributed_joblib.DaskDistributedBackend,
+                      joblib.parallel.ParallelBackendBase)
+
+
+def add5(a, b, c, d=0, e=0):
+    return a + b + c + d + e
+
+
+class CountSerialized(object):
+    def __init__(self, x):
+        self.x = x
+        self.count = 0
+
+    def __add__(self, other):
+        return self.x + getattr(other, 'x', other)
+
+    __radd__ = __add__
+
+    def __reduce__(self):
+        self.count += 1
+        return (CountSerialized, (self.x,))
+
+
+@pytest.mark.parametrize('joblib', joblibs)
+def test_joblib_scatter(loop, joblib):
+    if joblib is None:
+        pytest.skip()
+
+    Parallel = joblib.Parallel
+    delayed = joblib.delayed
+
+    x = CountSerialized(1)
+    y = CountSerialized(2)
+    z = CountSerialized(3)
+
+    with cluster() as (s, [a, b]):
+        with joblib.parallel_backend('dask.distributed', loop=loop,
+                                     scheduler_host=s['address'],
+                                     scatter=[x, y]) as (ba, _):
+            f = delayed(add5)
+            tasks = [f(x, y, z, d=4, e=5),
+                     f(x, z, y, d=5, e=4),
+                     f(y, x, z, d=x, e=5),
+                     f(z, z, x, d=z, e=y)]
+            sols = [func(*args, **kwargs) for func, args, kwargs in tasks]
+            results = Parallel()(tasks)
+
+            ba.terminate()
```

Comment:
Backends aren't contextmanagers, there is no `__exit__` to the backend. The call was already there, I moved it to use `terminate` which is more standard to the other joblib backends.